package.json

```diff
@@ -26,10 +27,14 @@
     "@trpc/react-query": "^10.45.2",
     "@trpc/server": "^10.45.2",
     "autoprefixer": "^10.4.19",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
+    "cmdk": "1.0.0",
     "country-flag-icons": "^1.5.12",
     "cva": "^1.0.0-beta.1",
     "drizzle-orm": "^0.31.2",
-    "lucide-react": "^0.396.0",
+    "husky": "^9.0.11",
```

Comment:
not needed. installed and handled automatically by lefthook

Reply:
Removed in 5d5ddc7d9fbf4410d79f73d65ad1bd91b54f079f.